code/modules/lootpanel/misc.dm

```diff
@@ -0,0 +1,39 @@
+/// Helper to open the panel
+/datum/lootpanel/proc/open(turf/tile)
+	source_turf = tile
+
+#if !defined(OPENDREAM) && !defined(UNIT_TESTS)
+	if(!notified)
+		var/build = owner.byond_build
+		var/version = owner.byond_version
+		if(build < 515 || (build == 515 && version < 1635))
+			to_chat(owner.mob, examine_block(span_info("\
+				<span class='bolddanger'>Your version of Byond doesn't support fast image loading.</span>\n\
+				Detected: [version].[build]\n\
+				Required version for this feature: <b>515.1635</b> or later.\n\
+				Visit <a href=\"https://secure.byond.com/download\">BYOND's website</a> to get the latest version of BYOND.\n\
+			")))
+
+			notified = TRUE
+#endif
+
+	populate_contents()
+	ui_interact(owner.mob)
+
+
+/// Used by SSlooting to process images from the to_image list.
```

Comment:
could you elaborate on what process means? I care less about where this proc is called and more about what it does